catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/RecyclerViewFragment.java

```diff
@@ -429,11 +430,21 @@ protected void showRenameDialog(List<T> selectedItems) {
 				.show();
 	}
 
-	protected void showMergeAlert() {
-		if (adapter.getSelectedItems().size() <= 1) {
+	protected void showMergeDialog(List<T> selectedItems) {
+		if (selectedItems.size() <= 1) {
 			ToastUtil.showError(getContext(), R.string.am_merge_error);
 		} else {
-			ToastUtil.showError(getContext(), "These two projects cannot yet be merged");
+			TextInputDialog.Builder builder = new TextInputDialog.Builder(getContext());
+
+			builder.setHint(getString(R.string.project_name_label))
```

Comment:
please group the setters:
builder.setText
.setHint
.setTextWatcher

builder.setPositiveButton
.setNegativeButton

builder.setTitle
builder.show.

or something similar...